docs/project_setup_and_build.md

```diff
@@ -0,0 +1,229 @@
+# Project structure and build
+
+## Overview
+
+In this chapter, you will get to know the anatomy of the standard BigFlow project, deployment artifacts, and [CLI commands](./cli.md).
+
+The BigFlow build command packages your processing code into a Docker image. Thanks to this approach, you can create any
+environment you want for your pipelines. What is more, you don't need to worry about dependencies clashes on the Cloud Composer (Airflow).
+
+The scheme of a BigFlow deployment artifacts looks like this:
+
+![BigFlow artifact scheme](./images/bigflow-artifact.png)
+
+Your project is turned into a standard Python package (which can be uploaded to [pypi](https://pypi.org/) or installed locally using `pip`). 
+Next, the package is closed into a Docker image with fixed Python version. Finally, there are Airflow DAGs that uses this image.
```

Comment:
there are Airflow DAGs **which** **use** this image

Reply:
done